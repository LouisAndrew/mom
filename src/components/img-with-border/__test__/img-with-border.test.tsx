import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import path from 'path';

import ImgWithBorder from '..';

describe('Img with border component!', () => {
    afterEach(cleanup);

    const img: React.ReactElement = (
        <img
            src="https://res.cloudinary.com/dsvdffre0/image/upload/v1596031267/w3ufcacls7mqfmhbwpe4.png"
            alt="lakers-logo"
        />
    );
    const el: React.ReactElement = <ImgWithBorder imgComponent={img} />;

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(el, div);
    });

    it('matches snapshot', () => {
        const tree = renderer.create(el).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
