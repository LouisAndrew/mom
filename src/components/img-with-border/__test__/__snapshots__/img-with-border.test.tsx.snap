// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Img with border component! matches snapshot 1`] = `
<div
  className="sc-AxjAm ePUdrK"
>
  <img
    alt="lakers-logo"
    src="../../../images/lakers-logo.png"
  />
</div>
`;
